settings.py

```diff
@@ -157,6 +157,9 @@
 # you can specify an array for multiple creative templates
 OPENWRAP_CREATIVE_TEMPLATE = 'ganeshformat'
 
+# option to set user-defined variable for NATIVE creative type (used only for NATIVE setup type)
```

Comment:
It will be used for In-app as well, right ?

Reply:
This variable is used only when the creative type is NATIVE, I have fixed the typo.